src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -57,14 +71,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
             new TableSeparator(),
             array('<info>Kernel</>'),
             new TableSeparator(),
-            array('Type', get_class($kernel)),
-            array('Name', $kernel->getName()),
```

Comment:
Please, don't remove the kernel name info, it is important for [Name-based Virtual Kernel](https://github.com/yceruto/symfony-skeleton-vkernel/blob/bdf53cedfde5e3e58709d6c18e99e3a71fbda192/src/VirtualKernel.php#L15) applications :(

Reply:
Agreed: please don't remove this output. Regardless, even if there was merit to the change, this seems outside the scope of this PR. We should avoid surprises that aren't directly related to the PR itself; otherwise, people will have difficulty even finding the discussion around the change.

Reply:
Well if we get rid of `Kernel::getName()` then we're good anyway :) 

Maybe i can look into that soonish; https://github.com/symfony/recipes/pull/186#issuecomment-331416924

In general i agree; if we keep it, it should be displayed here. Tend to remove Kernel type either way, you probably "know" this.

Reply:
Reverted for now :) should be discussed when deprecating getName.

Reply:
Let's not introduce something we know will be deprecated. If you really want to display something here, we could use the Kernel class name. That would help differentiate multi-kernel apps.

Reply:
We already show the kernel class name, no? Regardless, the issue isn't with *introducing* something, but that this PR *removed* something unrelated to the stated purpose of this PR. 

IMHO, the call to `getName()` here should be discussed (and possibly removed) when a PR to remove the `Kernel::getName()` method is introduced, not in a PR adding Dotenv output. Right?